### ISSUE TITLE
Category / Project information missing from JSON source for CP Depts

### ISSUE THREAD

**Issue by jimafisk** [CityOfBoston/budget.boston.gov#252]
Some of the CP Depts pages are missing their category / project information

![broken_cp_depts](https://user-images.githubusercontent.com/5913244/36605451-c12bc13e-188e-11e8-99f0-22654887f862.png)

Examples:
- https://budget.digital-staging.boston.gov/capital-projects/neighborhood-services/
- https://budget.digital-staging.boston.gov/capital-projects/administration-and-finance/
- https://budget.digital-staging.boston.gov/capital-projects/assessing-department/

**Comment by paulparisi** [comment]
This matches the live site https://budget.boston.gov/capital-projects/neighborhood-services/ - please advise how it should be different?

**Comment by jimafisk** [comment]
Good catch, I assume there are no initiatives or active projects for these CP Depts? I can look into hiding the titles / style if null.

**Comment by paulparisi** [comment]
I do not know that definitively - maybe someone from OBM could verify.

**Comment by jimafisk** [comment]
@jimwilliamson-boston can you take a look ^^^

Thanks!

**Comment by jimwilliamson-boston** [comment]
Is this a duplicate with the one I entered?  I will close mine.

**Comment by JackHanlon** [comment]
Not all departments have capital projects. The three depts Jim highlighted ONS, A&F, and Assessing do not have capital projects.
